airflow/executors/base_executor.py

```diff
@@ -150,16 +150,25 @@ def heartbeat(self) -> None:
         self.log.debug("Calling the %s sync method", self.__class__)
         self.sync()
 
-    def trigger_tasks(self, open_slots: int) -> None:
+    def order_queued_tasks_by_priority(self) -> List[Tuple[TaskInstanceKeyType, QueuedTaskInstanceType]]:
```

Comment:
What do you think about making this a property?

Reply:
This could make it harder to understand the code.  Note that a loop is used below that iterates over the result of this function, but at the same time modifies self.queued_tasks). If we create a property here, it will be difficult to see that property and self.queued_task are related.